spec/features/requester_edits_ncr_work_order_spec.rb

```diff
@@ -7,10 +7,11 @@
 
   let(:work_order) { create(:ncr_work_order, description: 'test') }
   let(:ncr_proposal) { work_order.proposal }
-  let!(:approver) { create(:user) }
+  let!(:approver) { create(:user, client_slug: 'ncr') }
```

Comment:
maybe we should make `ncr` a constant? Seems like we are using it all over the place and is worthy of constantization